app/controllers/conferences_controller.rb

```diff
@@ -12,16 +15,46 @@ def show
     @conference = Conference.find(params[:id])
   end
 
+  def create
+    @conference = Conference.new(conference_params)
+    generate_gid(@conference)
+    @team = Team.find(params[:team_id])
+
+    respond_to do |format|
+      if @conference.save
+        format.html { redirect_to params[:redirect_to].presence || edit_team_path(@team), notice: 'Conference was successfully created.' }
+        format.json { render action: :edit, status: :created, location: @team }
```

Comment:
I don't think we need to respond to json in this action. We don't have an API client (yet :wink: )